tests/installation/select_patterns_and_packages.pm

```diff
@@ -65,8 +65,11 @@ sub gotopatterns {
         send_key 'ret';
     }
 
-    if (check_screen('dependancy-issue', 10) && get_var("WORKAROUND_DEPS")) {
-        $self->record_dependency_issues;
+    if (check_screen('dependancy-issue') && get_var("WORKAROUND_DEPS")) {
+        $self->workaround_dependency_issues;
+    }
+    if (check_screen('dependancy-issue') && get_var("BREAK_DEPS")) {
```

Comment:
Better avoid check_screen with a timeout of default or lower but use multi-tag assert_screen in combination with match_has_tag